components/icon/Icon.tsx

```diff
@@ -10,40 +10,41 @@ import React, { useMemo } from 'react';
 import './icon.css';
 import { EIConTypes } from './icon.enum';
 import { IIconProps } from './icon.interface';
-import MumbleIcon from './svg/mumble.svg';
-import ArrowLeftIcon from './svg/arrow-left.svg';
-import ArrowRightIcon from './svg/arrow-right.svg';
-import ArrowUpIcon from './svg/arrow-up.svg';
-import ArrowDownIcon from './svg/arrow-down.svg';
-import CalendarIcon from './svg/calendar.svg';
-import CancelIcon from './svg/cancel.svg';
-import CheckmarkIcon from './svg/checkmark.svg';
-import CommentBorderedIcon from './svg/comment-bordered.svg';
-import CommentFilledIcon from './svg/comment-filled.svg';
-import EditIcon from './svg/edit.svg';
-import EyeIcon from './svg/eye.svg';
-import FullScreenIcon from './svg/full-screen.svg';
-import HeardBorderedIcon from './svg/heart-bordered.svg';
-import HeartFilledIcon from './svg/heart-filled.svg';
-import LocationIcon from './svg/location.svg';
-import LogoutIcon from './svg/logout.svg';
-import ProfileIcon from './svg/profile.svg';
-import RepostIcon from './svg/repost.svg';
-import SendIcon from './svg/send.svg';
-import SettingsIcon from './svg/settings.svg';
-import ShareIcon from './svg/share.svg';
-import TimeIcon from './svg/time.svg';
-import UploadIcon from './svg/upload.svg';
+import mumbleIcon from './svg/mumble';
```

Comment:
Why did you change the names and do not return React components anymore? Are there any benefits with this approach?

Reply:
I was not able to get rid of the type errors. I invested a lot of time trying to fix it but then I did not want to lose more time. We can try again later on but the functionality would be exactly the same. 
+ here. no needed dependency nor extra webpack config.

Reply:
That is good to me. Thx for the explanation